tools/challenge-parser/parser/plugins/add-video-question.js

```diff
@@ -6,40 +6,52 @@ const { splitOnThematicBreak } = require('./utils/split-on-thematic-break');
 
 function plugin() {
   return transformer;
-
   function transformer(tree, file) {
     const questionNodes = getAllBetween(tree, '--question--');
     if (questionNodes.length > 0) {
       const questionTree = root(questionNodes);
+      const assignmentNodes = getAllBetween(questionTree, '--assignment--');
```

Comment:
It would be better to keep these plugins nice and modular.  Rather than modifying how `add-video-question` works, I recommend creating an `add-assignment` plugin just for that.

The assignments and questions really are distinct, so they should not be bundled together by the parser.

Reply:
I think adding a whole other plugin for this is overkill, they video question have a lot in common with The Odin Project.